frontend/src/components/Header/Header.tsx

```
@@ -41,13 +57,28 @@ function Header() {
+            Services
+          </a>
+          <a href="#" className="block py-2">
+            Contact
+          </a>
+          {currentUser ? (
+            <Avatar>
+              <AvatarImage src="https://github.com/shadcn.png" alt="@shadcn" />
+              <AvatarFallback>{currentUser.firstName}</AvatarFallback>
+            </Avatar>
+          ) : (
+            <button className="w-full mt-2 px-4 py-2 dark:bg-gray-600 dark:hover:bg-gray-700 bg-gray-800 hover:bg-gray-900 text-white rounded-md">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)